Bottom: 59fe858d4fc5625a6a9b8a816691d191e7f3ad92
Top:    b9d9b229bba5c175ae092b3c3dc22c57820b73d9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-24 00:43:17 +0000

find.c: Alter LookupConnection to lookup inc./excl. rat-connections in one pass


---

diff --git a/src/find.c b/src/find.c
index 32122d3..7cedf79 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3047,8 +3047,7 @@ ListStart (int type, void *ptr1, void *ptr2, void *ptr3, int flag)
  * also the action is marked as undoable if AndDraw is true
  */
 void
-LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
-                  bool AndRats)
+LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int connected_flag, int found_flag)
 {
   void *ptr1, *ptr2, *ptr3;
   char *name;
